src/util/colorScale.ts

```diff
@@ -109,7 +117,11 @@ export const calcColorScale = (colorBy, controls, tree, treeToo, metadata) => {
     };
   } catch (err) {
     /* Catch all errors to avoid app crashes */
-    console.error("Error creating color scales. Details:\n", err.message);
+    if (err instanceof Error) {
+      console.error("Error creating color scales. Details:\n", err.message);
+    } else {
+      console.error("Unknown error creating color scales. Details:\n", err);
+    }
```

Comment:
I've never seen non-Error-derived expressions thrown in Auspice, although I know it's technically possible. I'm not thrilled with this verbosity / conditional error messaging spreading throughout the code base so how about 
```ts
  } catch (err: any) {
    /* Catch all errors to avoid app crashes */
    console.error("Unknown error creating color scales. Details:\n", err?.message);
```

Reply:
I think we should use `?` sparingly and am similarly not thrilled with its use spreading throughout the codebase.

In this case, `err?.message` would resolve to `undefined` instead of a potentially more meaningful value coming from `err` itself.

Would this still be too verbose?

```ts
} catch (err: unknown) {
  /* Catch all errors to avoid app crashes */
  const errorMessage = err instanceof Error ? err.message : String(err);
  console.error("Error creating color scales. Details:\n", errorMessage);
```

Reply:
Done: https://github.com/nextstrain/auspice/blob/ad1f2cedbde4df6974412d37fd84119208902065/src/util/colorScale.ts#L118-L121